scripts/run_backend_tests.py

```diff
@@ -110,10 +122,18 @@
     '--test_path',
     help='optional subdirectory path containing the test(s) to run',
     type=python_utils.UNICODE)
+_EXCLUSIVE_GROUP.add_argument(
+    '--test_shard',
+    help='optional name of shard to run',
+    type=python_utils.UNICODE)
```

Comment:
WHy unicode, are we expecting it to be an integer?

Reply:
JSON keys are strings, not integers, so shards are named as strings too. Those happen to be made of numeric characters, but that was arbitrary